index.bs

```diff
@@ -1144,8 +1144,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     :: The [=base64url encoding=] of |options|.{{PublicKeyCredentialRequestOptions/challenge}}
     : {{CollectedClientData/origin}}
     :: The [=ascii serialization of an origin|serialization of=] |callerOrigin|.
-    : {{CollectedClientData/tokenBindingId}}
-    :: The [=Token Binding ID=] associated with |callerOrigin|, if one is available.
+    : {{CollectedClientData/tokenBinding}}
+    :: The status of [=Token Binding=] between the client and the |callerOrigin| as well as the [=Token Binding ID=] associated with |callerOrigin|, if one is available.
```

Comment:
Tiny nit: I suggest moving the comma to just before "as well": "The status of [...], as well as the [...] if one is available". So that "if one is available" refers to just the token binding ID instead of the whole structure. You're welcome to ignore this comment if you wish.

Reply:
(Likewise.)